clients/src/main/java/org/apache/kafka/clients/Metadata.java

```diff
@@ -264,6 +267,13 @@ public synchronized long lastSuccessfulUpdate() {
     }
 
     /**
+     * The max allowable age of metadata.
+     */
+    public long maxAge() { 
+        return this.metadataExpireMs; 
```

Comment:
`metadata.maxAgeMs` makes sense to me. However, `metadata.maxMetadataAgeMs` sounds redundant.